org.tuxemon.Tuxemon.yaml

```diff
@@ -0,0 +1,66 @@
+---
+app-id: org.tuxemon.Tuxemon
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: org.tuxemon.Tuxemon
+rename-icon: icon
+finish-args:
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+# everything is stored in the .tuxemon folder
+# also the save files
+- --persist=.tuxemon
+- --share=network
+- --socket=pulseaudio
+- --device=dri
+modules:
+  # needed by pygame
+  # otherwise an error message occurs which complains about setuptools
+  - name: portmidi
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/PortMidi/portmidi.git
+        tag: v2.0.3
+
+  # needed by the rumble manager
+  - name: libshake
+    buildsystem: simple
+    build-commands:
+      - BACKEND=LINUX PREFIX=${FLATPAK_DEST} make install
+    sources:
+      - type: git
+        url: https://github.com/zear/libShake.git
+        tag: v0.3.2
+
+  - python3-requirements.json
+  - name: tuxemon
+    buildsystem: simple
+    build-commands:
+      - install -d /app/share
+      - install -d /app/bin
+      # lowercase tuxemon is used to search plugin module paths.
+      # So if using that name, no modules are found.
+      - install -d /app/share/Tuxemon
+      - cp -r tuxemon /app/share/Tuxemon
+      - cp -r mods /app/share/Tuxemon/mods
+      - cp run_tuxemon.py /app/share/Tuxemon/org.tuxemon.Tuxemon.py
+      - ln -s ../share/Tuxemon/org.tuxemon.Tuxemon.py /app/bin
+      - install org.tuxemon.Tuxemon /app/bin
+    post-install:
+      - desktop-file-install --dir=/app/share/applications buildconfig/flatpak/org.tuxemon.Tuxemon.desktop
+      - install -Dm755 buildconfig/flatpak/org.tuxemon.Tuxemon.appdata.xml /app/share/metainfo/org.tuxemon.Tuxemon.appdata.xml
+      - install -Dm755 mods/tuxemon/gfx/icon.png /app/share/icons/hicolor/64x64/apps/icon.png
+      - install -Dm755 mods/tuxemon/gfx/icon_32.png /app/share/icons/hicolor/32x32/apps/icon.png
```

Comment:
the lack of 128x128 (or scalable) icon will be a problem.

Reply:
@bitcraft do you have a 128x128 pixel icon?

Reply:
no, but i will ask around for suggestions

Reply:
@bitcraft should I create a temporary 128x128 icon? I created a patch for the pending pullrequest on Tuxemon. This will be removed, once the mr is merged and a new version of Tuxemon is created